charts/intel-gpu-resource-driver/templates/gpu-resource-driver.yaml

```
@@ -0,0 +1,235 @@
+  name: intel-gpu-params
+spec:
+  shared: false
+---
+apiVersion: resource.k8s.io/v1alpha2
+kind: ResourceClass
+metadata:
+  name: intel-gpu
+driverName: gpu.resource.intel.com
+parametersRef:
+  apiGroup: gpu.resource.intel.com/v1alpha2
+  kind: GpuClassParameters
+  name: intel-gpu-params
+
+---
+apiVersion: gpu.resource.intel.com/v1alpha2
+kind: GpuClassParameters
+metadata:
+  name: intel-gpu-monitor-params
+spec:
+  monitor: true
+  shared: false
+---
+apiVersion: resource.k8s.io/v1alpha2
+kind: ResourceClass
+metadata:
+  name: intel-gpu-monitor
+driverName: gpu.resource.intel.com
+parametersRef:
+  apiGroup: gpu.resource.intel.com/v1alpha2
+  kind: GpuClassParameters
+  name: intel-gpu-monitor-params
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: defaults
+  namespace: {{ include "intel-gpu-resource-driver.namespace" . }}
+data:
+# It is possible to override driver hardcoded defaults:
+# - resourceClaimParameters:
+#     if resource claim did not include any parameters, these will be used
+# - vf-memory
+#     per product amount of local memory, in MiB, the auto-added VFs will get
+  resourceClaimParameters.config: |
+    count=1
+    type=gpu
+    shareable=true
+  vf-memory.config: |
+    {
+      "max1550": 16385,
+      "max1450": 16384,
+      "max1100": 8192,
+      "flex140": 2048,
+      "flex170": 4096,
+    }
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "intel-gpu-resource-driver.name" .}}-kubelet-plugin
+  namespace: {{ include "intel-gpu-resource-driver.namespace" . }}
+  labels:
+    {{- include "intel-gpu-resource-driver.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "intel-gpu-resource-driver.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.kubeletPlugin.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+       {{- include "intel-gpu-resource-driver.templateLabels" . | nindent 8 }}
+    spec:
+      serviceAccount: intel-gpu-resource-driver-service-account
+      serviceAccountName: {{ include "intel-gpu-resource-driver.serviceAccountName" . }}
+      initContainers:
+      - name: init
+        image: {{ include "intel-gpu-resource-driver.fullimage" . }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command: ["/gas-status-updater", "--status", "NotReady"]
+        env:
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        securityContext:
+          {{- toYaml .Values.kubeletPlugin.containers.init.securityContext | nindent 10 }}
+      containers:
+      - name: kubelet-plugin
+        image: {{ include "intel-gpu-resource-driver.fullimage" . }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command: ["/kubelet-plugin", "-v", "5"]
```

```suggestion
        command: ["/kubelet-plugin"]
```
Let's not enable such verbosity by default.